src/ngx_http_lua_worker.c

```diff
@@ -130,4 +130,27 @@ ngx_http_lua_ffi_worker_count(void)
 
     return (int) ccf->worker_processes;
 }
+
+
+int
+ngx_http_lua_ffi_worker_type(void)
+{
+    return ngx_process;
+}
+
+
+void
+ngx_http_lua_ffi_worker_privileged(int enable)
```

Comment:
We should use verbs in such function names. `ngx_http_lua_ffi_enable_privileged_agent` is better here.